sdk/containerregistry/Azure.Containers.ContainerRegistry/src/autorest.md

```diff
@@ -3,16 +3,23 @@
 Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
-tag: package-2019-08
-require:
-    -  https://github.com/Azure/azure-rest-api-specs/tree/255757f41275e8ec474361690ea8886cae8a503b/specification/containerregistry/data-plane/readme.md
+input-file:
+    -  https://github.com/Azure/azure-sdk-for-js/blob/f3a5137513bfe904eeaddcd2d1d73191da90a63c/sdk/containerregistry/container-registry/swagger/containerregistry.json
```

Comment:
Should we be using the [readme.md](https://github.com/Azure/azure-sdk-for-js/blob/f3a5137513bfe904eeaddcd2d1d73191da90a63c/sdk/containerregistry/container-registry/swagger/README.md) file instead?

Reply:
Thank you - I had missed that this was there.

Reply:
The readme in the JS repo is currently TS-specific.  We'll look at generalizing it to all languages when we move these into the azure-rest-api-specs (discussing with @jeremymeng offline).  Thanks!